### PR TITLE
Add resources for 100 trillion digits

### PR DESCRIPTION
**Pull request by yuryu**
Adding resources for 100 trillion digits

- New index for the API
- Updated page content

### CODE REVIEW

File: functions_test.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func TestRest_Get(t *testing.T) {
 		{16, 1, 1, "2"},
 		{16, 0, 50, "3243f6a8885a308d313198a2e03707344a4093822299f31d00"},
 		{16, 1, 50, "243f6a8885a308d313198a2e03707344a4093822299f31d008"},
-		{16, 41_524_101_186_051, 100, "717a7a8ddd2bac9e3f80609daacc0580794ca7ec015"},
+		{16, 41_524_101_186_051, 100, "717a7a8ddd2bac9e3f80609daacc0580794ca7ec01574c4c8209871b599d3548d16e177cb52cbbbe26f621b522b3e6bf1845"},
```
[the remark this answers is not in the document]
Comment (yuryu) [reply functions_test.go:48]: Originally the API didn't return anything beyond 41,524,101,186,101, but now it does with the new calculation result, so I needed to add more digits.

File: jekyll/_demos/genie.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ position_number: 1
   Genie to map to the next logical note for great sounding Pi.
 </p>
 <p>
-  With 50 trillion digits and playing at 314 BPM, it'll play for 302 758.898
+  With 100 trillion digits and playing at 314 BPM, it'll play for 605 516
```
[the remark this answers is not in the document]
Comment (yuryu) [reply jekyll/_demos/genie.md]: done

File: jekyll/_introduction/about.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+position_number: 2
+---
+
+This website serves pi and interactive demos that you can listen to or see pi with.
+Scroll down to play with the demos!
+
+To learn more about how this site works, check out the [Serving the Pi](#howserving) section.
+We've also published the entire [source code](https://github.com/googlecloudplatform/pi-delivery) of the demos and infrastructure, including Terraform scripts, on GitHub.
+
+This website is not an official Google product. It is managed by volunteers of the [Google Cloud Developer Advocacy team](https://cloud.google.com/developers/advocates) and may go down or shut down without notice.
```
[the remark this answers is not in the document]
Comment (yuryu) [reply jekyll/_introduction/about.md]: changed to "become available"

File: jekyll/_how/who.md [hunk 4]
```diff
@@ -9,7 +9,9 @@ Since 2015, different members of the team worked on different Pi Day demos!
 
 In 2022, [Sara Ford](https://cloud.google.com/developers/advocates/sara-ford) and [Ben Bleything](https://twitter.com/bleything) [calculated 64 million digits of π](https://cloud.google.com/blog/topics/developers-practitioners/celebrating-pi-day-cloud-functions) using [Cloud Functions (2nd gen)](https://cloud.google.com/functions/docs/2nd-gen/overview). [Emma Haruka Iwao](https://cloud.google.com/developers/advocates/emma-haruka-iwao) rebuilt the API with Cloud Functions and the demos with React. We also made the source code available on [GitHub](https://github.com/GoogleCloudPlatform/pi-delivery).
```
[the remark this answers is not in the document]
Comment (yuryu) [reply jekyll/_how/who.md]: added

**Review by bleything — APPROVED** [review]
Thanks for making those changes, this looks great.

**Comment by bleything** [comment]
> Thank you, Ben. I agree the who section is a bit hard to follow. Can we merge this first and then start a new PR to change the structure?

Definitely. I'll put something together after this is merged.
